Templates/AssetGem/template.json

```diff
@@ -1,38 +1,38 @@
 {
     "template_name": "AssetGem",
-    "origin": "The primary repo for AssetGem goes here: i.e. http://www.mydomain.com",
-    "license": "What license AssetGem uses goes here: i.e. https://opensource.org/licenses/MIT",
-    "display_name": "AssetGem",
-    "summary": "A short description of AssetGem template.",
-    "canonical_tags": [],
+    "template_restricted_platform_relative_path": "Templates/AssetGem",
+    "origin": "Open 3D Engine - o3de.org",
+    "origin_url": "https://github.com/o3de/o3de",
+    "license": "Apache-2.0 or MIT",
+    "license_url": "https://github.com/o3de/o3de/blob/development/LICENSE.txt",
+    "display_name": "Asset Gem Template",
+    "summary": "Use this gem template to create the minimal gem needed for adding asset(s).",
+    "canonical_tags": [
+		"Template",
+        "Gem",
+        "Asset"
```

Comment:
nit: alignment issue here

Reply:
looks like a tab vs spaces. Should be ok now.